webui/react/src/types.ts

```diff
@@ -575,6 +579,11 @@ export interface ExperimentItem {
   workspaceName?: string;
 }
 
+export interface ExperimentWithTrial {
+  experiment?: ExperimentItem;
+  bestTrial?: TrialItem;
+}
```

Comment:
if we do `type ExperimentWithTrial = ExperimentItem & TrialItem`-- while also renaming `trial.id` to `trialId`, we wont have to deal with the additional substructure, but i think there are good arguments for both